docs/networking.adoc

```diff
@@ -118,6 +116,8 @@ fd00:1122:3344:105::/64     fe80::aa40:25ff:fe05:41c    UG      1       0 cxgbe1
 
 We'll talk about this more later.
 
+Finally, you can use the portable `ifconfig` tool to list much of the same information.  We'll be sticking to the illumos tools.
```

Comment:
```suggestion
Finally, if you're used to using `ifconfig` to look at this information, you can do that on illumos too. However, we prefer using `ipadm` for managing this and stick to it throughout this document.
```

I put this here mostly because ifconfig is only kind of portable and I still think of it as an illumos tool. Parts of it are the same between systems, but not everything. Dunno. Feel free to take or leave.